bridge/engine/EngineStreamDirector.h

```diff
@@ -29,20 +29,24 @@ class EngineStreamDirector
         utils::Optional<SimulcastStream> _secondary;
         size_t _highestEstimatedPinnedLevel;
         size_t _desiredHighestEstimatedPinnedLevel;
+        size_t _desiredUnpinnedLevel;
```

Comment:
You need to initialise all this values and you haven't change the `makeParticipantStreams`, then there are uninitialised values that drives to undefined behaviours. Please add a constructor here to increase the code safety (initilize the default values and receive what is required to make a new object)